package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ainblockchain/ain-util",
-  "version": "1.1.6",
+  "version": "1.1.7",
```

Comment:
do we need a new version for the changes?

Reply:
As version up was not made in the prev updates, I updated it now. Thanks for the question!